libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -58,6 +63,9 @@
 
     private WPAndroidGlueCode mWPAndroidGlueCode;
 
+    private ConcurrentHashMap<String, Float> mUploadingMediaProgressMax = new ConcurrentHashMap<>();
+    private Set<String> mFailedMediaIds = new HashSet<>();
```

Comment:
Any reason to use interface `Set` instead of `HashSet` ?

Reply:
not really, just copied the code to represent the same functionality we had as parity with Aztec from here https://github.com/wordpress-mobile/WordPress-Android/blob/develop/libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java#L172